resources/lib/util/moonlighthelper.py

```diff
@@ -59,20 +59,21 @@ def list_games(self):
         list_regex = r'\d+\.\s+([^\n]*)'
 
         try:
-            moonlightOut = subprocess.check_output(['moonlight', 'list'], cwd=binary_path, timeout=5, encoding='utf-8', start_new_session=True)
+            host = self.plugin.getSetting('host_addr')
+            moonlightOut = subprocess.check_output(['moonlight', 'list', host], cwd=binary_path, timeout=5, encoding='utf-8', start_new_session=True)
```

Comment:
This adds the configured host address as argument to moonlight. Without it, Luna would err.

Reply:
I think this is because I don't have any [mDNS](https://github.com/moonlight-stream/moonlight-embedded/blob/master/moonlight.conf#L2) configured on my network. Is this something we need to take into account?

Reply:
Looking more closely, I think this is because the written config file is not read. Luna writes `moonlight.conf` to one of these paths:
-  `/etc/moonlight.conf`
- The binary path that MLE is installed in (e.g. `/usr/local/bin/moonlight.conf`)
- If above aren't writable, it falls back to the addon path (e.g. `~/.kodi/addons/script.luna/moonlight.conf`).

According to [the docs](https://github.com/moonlight-stream/moonlight-embedded/blob/274d3db34da764344a7a402ee74e6080350ac0cd/docs/README.pod#config-file), MLE only tries to read from the first one by default or from `$XDG_CONFIG_HOME/moonlight/moonlight.conf`. 

So I think I need to change this to either pass the config path explicitly, or write the config file to `$XDG_CONFIG_HOME/moonlight/moonlight.conf` always. What do you think?